source/locationHelper.py

```diff
@@ -0,0 +1,398 @@
+			try:
+				other=toPoint(other)
+			except ValueError:
+				return False
+		return (self.y, self.x) >= (other.y, other.x)
+
+	def __eq__(self,other):
+		if not isinstance(other,POINT_CLASSES):
+			try:
+				other=toPoint(other)
+			except ValueError:
+				return False
+		return self.x==other.x and self.y==other.y
+
+	def __neq__(self,other):
+		return not (self==other)
+
+	def toPOINT(self):
+		"""Converts self to a L{ctypes.wintypes.POINT}"""
+		return POINT(self.x,self.y)
+
+	def toLogical(self, hwnd):
+		"""Converts self from physical to logical coordinates and returns a new L{Point}."""
+		return Point(*windowUtils.physicalToLogicalPoint(hwnd, *self))
+
+	def toPhysical(self, hwnd):
+		"""Converts self from logical to physical coordinates and returns a new L{Point}"""
+		return Point(*windowUtils.logicalToPhysicalPoint(hwnd, *self))
+
+	def toClient(self, hwnd):
+		"""Converts self from screen to client coordinates and returns a new L{Point}"""
+		return Point(*winUser.ScreenToClient(hwnd, *self))
+
+	def toScreen(self, hwnd):
+		"""Converts self from client to screen coordinates and returns a new L{Point}"""
+		return Point(*winUser.ClientToScreen(hwnd, *self))
+
+class _RectMixin:
+	"""Mix-in class for properties shared between Location and Rect classes"""
+
+	def toRECT(self):
+		"""Converts self to a L{ctypes.wintypes.RECT}"""
+		return RECT(self.left,self.top,self.right,self.bottom)
+
+	def toLogical(self, hwnd):
+		left,top=self.topLeft.toLogical(hwnd)
+		right,bottom=self.bottomRight.toLogical(hwnd)
+		if isinstance(self, Location):
+			return Location(left,top,right-left,bottom-top)
+		return Rect(left,top,right,bottom)
+
+	def toPhysical(self, hwnd):
+		left,top=self.topLeft.toPhysical(hwnd)
+		right,bottom=self.bottomRight.toPhysical(hwnd)
+		if isinstance(self, Location):
+			return Location(left,top,right-left,bottom-top)
+		return Rect(left,top,right,bottom)
+
+	def toClient(self, hwnd):
+		left, top =self.topLeft.toClient(hwnd)
+		if isinstance(self, Location):
+			return Location(left, top, self.width, self.height)
+		return Rect(left, top, left+self.width, top+self.height)
+
+	def toScreen(self, hwnd):
+		left,top=self.topLeft.toScreen(hwnd)
+		if isinstance(self, Location):
+			return Location(left, top, self.width, self.height)
+		return Rect(left, top, left+self.width, top+self.height)
+
+	@property
+	def topLeft(self):
+		return Point(self.left,self.top)
+
+	@property
+	def bottomRight(self):
+		return Point(self.right,self.bottom)
+
+	@property
+	def center(self):
+		return Point((self.left+self.right)/2, (self.top+self.bottom)/2)
+
+	def __contains__(self,other):
+		"""Returns whether other is a part of this rectangle."""
+		if isinstance(other,POINT_CLASSES):
+			return other.x >= self.left < self.right and other.y >= self.top < self.bottom
+		if isinstance(other,RECT_CLASSES):
+			return self>other
+		try:
+			other=toRect(other)
+		except:
+			return False
+		return self>other
+
+	def __lt__(self,other):

```

Comment:
Again, for the same arguments with point, I don't think operator overriding is a clear way of providing this behaviour.